apps/docs/components/marketing/nextui-pro-section.tsx

```diff
@@ -156,25 +167,20 @@ export const NextUIProSection = () => {
         <div className="overflow-hidden">
           <Marquee
             vertical
-            className="h-78 hidden md:flex w-screen mt-4 md:absolute md:top-0 md:inset-0 isolate md:max-h-dvh"
-            duration={isMobile ? 240 : 60}
+            className="flex opacity-15 md:opacity-100 w-screen mt-4 absolute top-0 inset-0 isolate max-h-dvh "
+            duration={60}
           >
-            <img
-              alt="Hero Background"
-              className="w-full"
-              height={3255}
-              loading="lazy"
-              src={imgSrc}
-              width={1920}
-            />
+            <img alt="Hero Background" className="w-full opacity-0 animate-fadeIn" src={imgSrc} />
```

Comment:
@macci001 why don't we use Next.js Image component instead so we avoid preloading the images?

Reply:
Done, updated in latest commit.